debezium-server-kinesis/src/main/java/io/debezium/server/kinesis/KinesisChangeConsumer.java

```diff
@@ -55,6 +60,8 @@ public class KinesisChangeConsumer extends BaseChangeConsumer implements Debeziu
     private String region;
     private Optional<String> endpointOverride;
     private Optional<String> credentialsProfile;
+    private static final int DEFAULT_RETRIES = 5;
+    private static final Long RETRY_INTERVAL = Integer.toUnsignedLong(1_000);
 
```

Comment:
It would be interesting if the DEFAULT_RETRIES and RETRY_INTERVAL were config properties. Something like debezium.sink.kinesis.default.retries and debezium.sink.kinesis.retry.interval. WDYT?

Reply:
[CMIIW] I am not sure about this; it's technically possible, but in my opinion, it's not necessary. If we take a look at other sinks:

1. For HTTP, the retries are hardcoded to 5.
2. For EventHub, the client uses the default retries, which is 9 times.

The exception is Pub/Sub, where the retry properties can be dynamically set via properties.

Do you have any comment @vjuranek?